CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ Changes are grouped as follows
 
 ## Unreleased
 
+## [2.3.0] - 2020-08-25
+
+### Changed
+- Provide type information to users of this package
```

Comment:
Maybe something a bit more specific so users know about the new functionality.

```suggestion
- Add Mypy support by adding packaging type information
```

Something like that? Keep in mind I'm not a python developer

Reply:
Great suggestion!